test/unit/test/sandboxed_path_test.cpp

```diff
@@ -292,7 +292,7 @@ TEST(sandboxed_path_symbolic_link, symbolic_link)
     // if link already exists, remove it
     if (std::filesystem::exists(tmp_dir))
     {
-        std::filesystem::remove_all(tmp_dir);
+        std::filesystem::remove(tmp_dir);
```

Comment:
This would fail is tmp_dir already exists and is a non-symlink directory, right? Probably a quite unrealistic scenario, but should we cover it?